app/lib/config/config.go

```diff
@@ -237,3 +240,13 @@ func (s *ServerToml) String() string {
 	}
 	return buff.String()
 }
+
+// TildeToHome takes a path and replaces an eventual "~" with the home-directory
+func TildeToHome(path string) string {
```

Comment:
> For the ErrFatal, I can return an error, would that be better? Or just ignore the error and replace the ~ with a . if the user-directory is not available?

Sorry right, I'm used to that notation yet (no `if` testing!). Ok then indeed we have to fatal otherwise we can't do much, but maybe we can fatal with a more explicit error: here it will be `user.UnknownUserError` [link](https://golang.org/pkg/os/user/#Current) which is weird I think.